### PR TITLE
Feature: Authenticated CQRS requests to the server

### PR DESCRIPTION
**Pull request by Dragemil**

### CODE REVIEW

File: mobile/lib/features/cqrs/command_result.dart [hunk 1]
```diff
@@ -0,0 +1,14 @@
+abstract class CommandResult {}
+
+class SuccessCommandResult extends CommandResult {
```
Comment (BartoszChrostowski) [inline mobile/lib/features/cqrs/command_result.dart:3]: I assume every command should return id?

File: mobile/lib/features/cqrs/cqrs_client.dart [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import 'package:tree_of_a_kind/features/cqrs/cqrs_action.dart';
+
+class CqrsClient {
+  CqrsClient(this._apiUri, {FirebaseAuth firebaseAuth})
+      : assert(_apiUri != null),
+        _firebaseAuth = firebaseAuth ?? FirebaseAuth.instance;
+
+  final Uri _apiUri;
+  final FirebaseAuth _firebaseAuth;
+  final Duration timeout = const Duration(seconds: 30);
```
Comment (BartoszChrostowski) [inline mobile/lib/features/cqrs/cqrs_client.dart:16]: Magic number 😂🤣😂

File: mobile/lib/features/home/view/home_page.dart [hunk 3]
```diff
@@ -39,11 +39,11 @@ class _HomePageState extends State<HomePage> {
         child: Column(
           mainAxisSize: MainAxisSize.min,
           children: <Widget>[
-            Avatar(photo: user.photo),
+            Avatar(photo: user.photoURL),
```
Comment (BartoszChrostowski) [inline mobile/lib/features/home/view/home_page.dart:42]: Does the user have a photo?

**Review by BartoszChrostowski — APPROVED** [review]

File: mobile/lib/features/home/view/home_page.dart (hunk 3 above)
Comment (Dragemil) [reply mobile/lib/features/home/view/home_page.dart:42]: It's firebase user and he might photo from Google, not in our case though.

File: mobile/lib/features/cqrs/command_result.dart (hunk 1 above)
Comment (Dragemil) [reply mobile/lib/features/cqrs/command_result.dart:3]: No, if validation fails, command won't run thus no ID will be generated.

**Pull request merged**